senderWrapper.go

```diff
@@ -155,12 +143,6 @@ func (sw *CaduceusSenderWrapper) Update(list []ancla.InternalWebhook) {
 		sender, ok := sw.senders[inValue.ID]
 		if !ok {
 			osf.Listener = inValue.Listener
-			metricWrapper, err := newMetricWrapper(time.Now, osf.QueryLatency)
-
-			if err != nil {
-				continue
-			}
-			osf.ClientMiddleware = metricWrapper.roundTripper
```

Comment:
this was not a good place for this section, moved to outboundSenderFactory (factories like these will be removed in a later pr)